source-bigquery-batch/.snapshots/TestFeatureFlagKeylessRowID-Enabled-Capture

```diff
@@ -1,24 +1,22 @@
 # ================================
-# Collection "acmeCo/test/featureflagkeylessrowid_enabled_905157": 23 Documents
```

Comment:
I'm not sure why there'd be fewer documents in this snapshot after this change, especially since all the other snapshots are changing only in the expected way.

Reply:
The BigQuery test snapshots are just kind of unstable (because BigQuery is super slow and really goes hard on the "eventually your changes will show up" side of things, IIRC). They're still good enough to serve as a sanity check when run locally, but that's one of the reasons we don't run the BQ database tests in CI.